.github/workflows/triage.yml

```diff
@@ -0,0 +1,24 @@
+name: Take_Issue
+
+on:
+  issue_comment:
+    types:
+      - created
+      - edited
+  workflow_call:
+
+jobs:
+  take-issue:
+    name: Take issue
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    timeout-minutes: 10
+    steps:
+      - name: take an issue
+        uses: bdougie/take-action@main
+        with:
+          message: Thanks for taking this on! If you have not already, PM us
+          issueCurrentlyAssignedMessage: Thanks for being interested in this issue. It looks like this ticket is already assigned to a contributor. Please communicate with the assigned contributor to confirm the status of the issue.
```

Comment:
@mlukasze do you think we should limit the issues to be taken by a single person only? I think we could discuss whether we allow the possibility for multiple people working on a task. An example of that would be https://github.com/openvinotoolkit/openvino/issues/18347, which:
1. has been taken by a contributor
2. another contributor wanted to take it
3. the contributor got denied, since the task has already been assigned
4. the task (seemingly) got abandoned by the original picker

The other option would be to merge it as is and polish the process later.

Reply:
if assignees can coordinate work I see no problem with cooperating on complex tasks with more than 1 person.
but please be aware about risk of miscommunication to manage, that may slow down a work, so please be careful.

Reply:
I think we should allow for cooperation, in case there are issues with it we could revert back to having at most one assignee. @NsdHSO could you please allow for multiple assignees? I'll approve the PR soon.

Reply:
I tend to believe that in everyday life as a developer one cannot work on 2 tasks. But I think that whoever buys a ticket should be able to take it to the end. If we want this, even if a ticket is assigned to someone so that you can assign it to someone else, I should create an issue for the action we use. and the action to set a customization parameter. I have some pros and cons.
1. Collaboration and Communication: Allowing multiple contributors to work on a single task can foster collaboration and knowledge sharing. However, as mentioned, it might introduce challenges in terms of coordination and communication. Miscommunication could indeed slow down the work, as different contributors might approach the task differently.

2. Risk of Abandonment: If a task is assigned to a single contributor and they abandon it, having the ability to reassign it to someone else can help prevent the task from being stalled.

3. Ticket Ownership: The idea that the person who buys or claims a ticket should be able to take it to completion promotes a sense of ownership and responsibility. This can result in more accountable contributors.

4. Workflow Flexibility: Allowing multiple assignees can be useful for more complex tasks that require a variety of skills. It can also be beneficial during peak workloads when tasks need to be completed quickly.

5. Workflow Complexity: Allowing multiple assignees could add complexity to the process, especially if you need to manage concurrent changes and merge contributions.

Reply:
In this action, we check if the issue is assigned to somebody 
[action](https://github.com/bdougie/take-action/blob/main/action.yml#L37C10-L37C61)